src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1230,6 +1230,17 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		}
 	}
 
+	async copyLastOutput(): Promise<void> {
+		const cmdDetection = this.capabilities.get(TerminalCapability.CommandDetection);
+		const commands = cmdDetection?.commands;
+		if (commands && commands.length > 0) {
+			const output = commands[commands.length - 1]?.getOutput();
+			if (output) {
```

Comment:
`getOutput` will return `undefined` if the last command that was run is in the scrollback, so I don't think this will work for the case you're interested in

Reply:
It should only be if it's trimmed from the scrollback, so if the command decoration is there it should always have `getOutput()`.